src/constants.js

```diff
@@ -145,6 +145,11 @@ export const RGBADepthPacking = 3201;
 export const TangentSpaceNormalMap = 0;
 export const ObjectSpaceNormalMap = 1;
 
+// Color space string identifiers, matching CSS Color Module Level 4 and WebGPU names where available.
+export const NonColorData = '';
```

Comment:
We prefer `NonColorData` better than `NoColorSpace`?

Reply:
I've been staring at these terms too long, they all blur together now. I'm completely happy with either. NoColorSpace has a nice parallelism with NoToneMapping... Do you have a preference?

Reply:
I vote for `NoColorSpace` yeah.

Reply:
Sounds good — done: [68407d7](https://github.com/mrdoob/three.js/pull/23392/commits/68407d7c0a89eefdd2a187cccabdf0ed0cde892b)